app/controllers/alerts_list_controller.rb

```diff
@@ -2,6 +2,7 @@
 
 # Controller for showing Alert records
 class AlertsListController < ApplicationController
+  include AlertsHelper
```

Comment:
@sejalkhatri I think I was wrong. Please retract my previous comment. All I wanted to convey was that the helper methods matching the resource are automatically available in the views, not in the controllers.

You don't really want to pollute the `controller` by including the all the methods in the helper. In fact. So, I don't think rails will automatically include it for you unless you _ask_ it to by `including` it in the controller.

Someone correct if I am wrong.

> But I don't know why it still works for me when I am not including the helper module in the controller.

Because you're only accessing the helper methods in the views?